ispyb-update-all-proposals

```diff
@@ -37,17 +37,10 @@ def main(args=[]):
         nsls2api_lib.ispyb_instruments
     )
     print(f"Number of ISPyB proposals is {len(ispyb_proposals)}")
-    for proposal in ispyb_proposals:
+    for index, proposal in enumerate(ispyb_proposals, 1):
         # get all users in PASS proposal
-        print(f"handling proposal: {proposal}")
+        print(f"{index}: handling proposal: {proposal}")
         proposal_info = nsls2api_lib.get_proposal_info(proposal)
-        ispyb_lib.add_usernames_for_proposal(
```

Comment:
You removed this call in a couple places, why is that?

Reply:
This is because the following line contains within it a call to `add_usernames_for_proposal()` with the following call hierarchy:

1. `ispyb_lib.reset_users_for_proposal()`
2. `add_users_for_proposal()`
3. `add_usernames_for_proposal()`

https://github.com/NSLS2/nsls2api-client-for-mx/blob/3d124117ba76ef42e36d9167c23b4fd2a4af5461/ispyb_lib.py#L247-L254